redis/src/main/scala/zio/redis/TestExecutor.scala

```diff
@@ -27,6 +30,24 @@ private[redis] final class TestExecutor private (
             input.head
         }
 
+      case api.Connection.Auth.name =>
+        STM.succeedNow {
+          if (input.isEmpty) errResponse(api.Connection.Auth.name)
+          else
+            RespValue.bulkString("OK")
+        }
+      case api.Connection.Echo.name =>
+        STM.succeedNow {
+          if (input.isEmpty) errResponse(api.Connection.Echo.name)
+          else
+            input.head
+        }
+      case api.Connection.Select.name =>
+        STM.succeedNow {
+          if (input.isEmpty) errResponse(api.Connection.Select.name)
+          else RespValue.bulkString("OK")
+        }
```

Comment:
```suggestion
      case api.Connection.Auth.name =>
        STM.succeedNow(if (input.isEmpty) errResponse(name) else RespValue.bulkString("OK"))
        
      case api.Connection.Echo.name =>
        STM.succeedNow(if (input.isEmpty) errResponse(name) else input.head)
        
      case api.Connection.Select.name =>
        STM.succeedNow(if (input.isEmpty) errResponse(name) else RespValue.bulkString("OK"))
```

Optionally, you can extract a helper.